category.php

```diff
@@ -1,6 +1,15 @@
 <?php get_header(); ?>
 
 <?php
+$term = get_queried_object();
+
+$enable_custom_page_content = get_field( 'category_customize_page_content', $term );
+$custom_page_content = get_field( 'category_custom_page_content', $term );
+// Ensure that enable_sidebar is true if ACF setting has not been set yet
+$enable_sidebar = ( get_field( 'category_enable_sidebar', $term ) === null ) ? true : get_field( 'category_enable_sidebar', $term );
+// Ensure that customize_sidebar is false if ACF setting has not been set yet
+$customize_sidebar = ( get_field( 'category_customize_sidebar', $term ) === null ) ? false : get_field( 'category_customize_sidebar', $term );
+
 $posts = get_posts( array(
 	'numberposts' => 10,
 	'cat'         => get_queried_object()->term_id
```

Comment:
As a tiny enhancement, we could sub out the `get_queried_object()` call here and replace it with a reference to `$term`.